web/templates/admin/meal/index.html.eex

```diff
@@ -1,35 +1,28 @@
-<h2>Listing meals</h2>
+<div class="row divider">
+  <div class="col-sm-10">
+    <h2>Weekly Meals</h2>
+  </div>
+  <div class="col-sm-2">
+    <%= link "New meal", to: admin_meal_path(@conn, :new), class: "btn btn-primary" %>
+  </div>
+</div>
 
-<table class="table">
-  <thead>
-    <tr>
-      <th>Description</th>
-      <th>Date</th>
-
-      <th></th>
-    </tr>
-  </thead>
-  <tbody>
 <%= for meal <- @meals do %>
-    <tr>
-      <td><%= meal.description %></td>
-      <td>
-          <%= Timex.format!(
-              meal.date,
-              "%m-%d-%Y",
-              :strftime
+  <div class="row top-space row-bottom-border row-striped">
+    <div class="col-sm-5 larger-text">
+      <strong><%= link meal.description, to: admin_meal_path(@conn, :edit, meal) %></strong>
+    </div>
+    <div class="col-sm-4 larger-text">
+      <%= Timex.format!(
```

Comment:
I would recommend moving this to a function in your view that's a bit simpler. This is pretty involved code to be in the template.